crowbar_framework/app/assets/javascripts/installer.js

```diff
@@ -0,0 +1,86 @@
+$(document).ready(function() {
+  $("body.installer").each(function() {
+    function statusCheck() {
+      $.ajax({
+        type: "GET",
+        dataType: "json",
+        url: Routes.installer_status_path(),
+        success: function(data) {
+          var failed = data.failed;
+          var success = data.success;
+          var errorMsg = data.errorMsg
+          var successMsg = data.successMsg
+          var noticeMsg = data.noticeMsg
+          data = data.steps;
+          var mostRecent = $("li." + data[data.length-1]);
+          var mostRecentIcon = mostRecent.children();
+
+          $.each(data, function (index, value) {
+            $("li." + value)
+              .removeClass("list-group-item-success")
+              .children("span")
+              .removeClass("fa-hourglass-o fa-circle-o-notch fa-spin")
+              .addClass("fa-check");
+          });
+
+          mostRecentIcon.removeClass(function (index, css) {
+            var match = css.match(/^fa-/g);
+            if (match !== null) {
+              return match.join("");
+            }
+          });
```

Comment:
But why not do like what you did a few lines above: `.removeClass("fa-hourglass-o fa-circle-o-notch fa-spin")`? AFAICT, you're in the exact same case.